osu.Game/Graphics/Containers/UserDimContainer.cs

```diff
@@ -76,9 +76,9 @@ protected override void LoadComplete()
         /// </summary>
         protected virtual void UpdateVisuals()
         {
-            ContentDisplayed = ShowDimContent;
+            ContentDisplayed = !EnableUserDim.Value || ShowDimContent;
```

Comment:
You can probably move this to the default of `ShowDimContent`

Reply:
I think you forgot to delete this

Reply:
Whoops